src/uct/ib/base/ib_verbs.h

```diff
@@ -60,7 +60,14 @@
 
 #else
 #  define IBV_SHARED_MR_ACCESS_FLAGS(_shared_mr)    ((_shared_mr)->access)
+#ifndef HAVE_EXP_UMR_KSM
```

Comment:
minor: i think it's better to check for IBV_EXP_DEVICE_ATTR_RESERVED_2 explicitly